### PR TITLE
Renaming foreach variable to avoid name conflict with the possible input parameter named value

### PR DESCRIPTION
**Pull request by hpaknia**
To fix issue: https://github.com/OpenAPITools/openapi-generator/issues/4231

Ran following commands to regenerate C# files:
```
mvn clean isntall \
  && ./bin/csharp-dotnet2-petstore.sh \
  && ./bin/csharp-netcore-petstore-netcore.sh \
  && ./bin/csharp-petstore-all.sh \
  && ./bin/csharp-petstore-net-40.sh \
  && ./bin/csharp-petstore-netcore-project.sh \
  && ./bin/csharp-property-changed-petstore.sh \
  && ./bin/csharp-netcore-petstore-all.sh \
  && ./bin/csharp-netcore-petstore.sh \
  && ./bin/csharp-petstore-net-35.sh \
  && ./bin/csharp-petstore-net-standard.sh \
  && ./bin/csharp-petstore.sh
```

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

File: samples/client/petstore/R/test_petstore.bash [hunk 1]
```diff
@@ -4,7 +4,9 @@ set -e
 
 REPO=http://cran.revolutionanalytics.com
 
+export LC_ALL=en_US.UTF-8
 export R_LIBS_USER=$HOME/R
+export LANG=en_US.UTF-8
```
Comment (wing328) [inline samples/client/petstore/R/test_petstore.bash]: @hpaknia why do you need these changes?

Comment (hpaknia) [reply samples/client/petstore/R/test_petstore.bash]: @wing328 `ci/circleci` is failing. I'm trying to go around this error:
```
* checking package directory ... OK
* checking DESCRIPTION meta-information ... ERROR
npm WARN npm npm does not support Node.js v13.0.0
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 6, 8, 9, 10, 11, 12.
npm WARN npm You can find the latest version at https://nodejs.org/
* DONE

Status: 1 ERROR
See
  ‘/home/circleci/OpenAPITools/openapi-generator/samples/client/petstore/R/petstore.Rcheck/00check.log’
for details.

[ERROR] Failed to execute goal or
```
Anyhow, the commit did not fix the issue. I dropped the second commit. Can you help me with debugging this https://circleci.com/gh/OpenAPITools/openapi-generator/9752?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link?

Comment (wing328) [reply samples/client/petstore/R/test_petstore.bash]: The issue has fixed itself by CircleCI (likely they reverted a bad change in their side)

Latest is green: https://circleci.com/gh/OpenAPITools/openapi-generator/9767

**Comment by hpaknia** [comment]
Awesome, will you be able to move this PR foreard? We are eagerly waiting
for the fix.

**Comment by wing328** [comment]
@hpaknia thanks for the PR. We can go with your fix. 

I may file another PR to rename more local variables with the localVar prefix

**Comment by hpaknia** [comment]
Thanks @wing328, is the docker image and the java jar file released automatically?

**Comment by wing328** [comment]
Yes, a snapshot version will be published as part of the CI/CD workflow shortly (~ 30 to 60 mins)

**Pull request merged**